### ISSUE TITLE
Unterstützer darstellen

### ISSUE THREAD

**Issue by jessor** [freifunk-fulda/fulda.freifunk.net#1]

**Comment by mfuhrmann** [comment]
Welches Format wollen wir nehmen? Liste, Tabelle, Visitenkartenformat?
Wichtiger Inhalt ist eigentlich nur der Firmenname und eine URL. Ggfs. ein Logo?

**Comment by jessor** [comment]
Wieviele sind das denn? Gibt es schon eine Liste?

Ich könnte mir das mit Logo und Link im Footer vorstellen. Evtl. Logo ausgegraut und in voller Farbe bei Hover.

![freifunk-sponsors](https://cloud.githubusercontent.com/assets/480451/10417262/b2aa8fae-7035-11e5-9799-61ce405b693c.png)

**Comment by mfuhrmann** [comment]
Ja. Ich hatte mal ein simples Listing angefangen. https://fulda.freifunk.net/freifunk-unterstuetzer

edit: Ich würde die Firmen alle mal anschreiben, mit der Bitte über das Konaktformular ein OK zu geben. Und sofern gewünscht auch ein Logo zu übermitteln.

**Comment by SISheogorath** [comment]
:+1: ich finde die Footeridee nice. Klar, die Firmen finden dann bestimmt "Könnte das nicht oben stehen?", aber ich würde sagen, es ist ein oft genutztes Schema und würde ich so unterstützen. 

Man sollte allerdings (vielleicht auch erst bei hover) einfach noch den Namen drunter einblenden.

**Comment by mfuhrmann** [comment]
Also zu viel Arbeit würde ich nicht da rein stecken. Es gibt auch jetzt schon Firmen, die kein Logo haben. Was passiert mit denen?

**Comment by jessor** [comment]
@reissmann Welche Unterstützer sollen denn erstmal genannt werden?

**Comment by SISheogorath** [comment]
Ich kläre das mal bis Freitag ab

**Comment by jessor** [comment]
Liste wird hier geführt: https://pad.freifunk.net/p/fffd-unterstuetzer
